packages/terra-application-layout/docs/about.md

```diff
@@ -0,0 +1,19 @@
+# Terra Application Layout
+
+The Terra Application Layout is a responsive, themeable layout for building applications.
+
+The Terra Application Layout provides:
+- A themeable, responsive application header, with APIs for rendering application-specific branding, tabular navigation, and user-centric utility controls.
```

Comment:
Should we specify the use of terra-utility?

Reply:
Yeah, that'd be more consistent.